src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SORTKEY;
+
+import java.util.Comparator;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Reorders all persons in TAlent Assistant™ based on the sort key provided.
+ * The sort key will point to a specific candidate field.
+ */
+public class SortCommand extends Command {
```

Comment:
Very clean and neat codes! Good that the java docs/comments are present, also following the coding standard closely! 👍🏻